### ISSUE TITLE
Drawer: `open` field is ignored

### ISSUE THREAD

**Issue by j340m3** [aforemny/material-components-web-elm#79]
Hello, I'm trying to integrate this library in my project, but I'm having trouble creating a `modalDrawer` or `dismissibleDrawer`. 

Here is my code

```import Html exposing (text)
import Html.Attributes exposing (style)
import Material.Drawer
    exposing
        ( drawerContent
        , permanentDrawer
        , permanentDrawerConfig
        , modalDrawer
        , modalDrawerConfig
        , dismissibleDrawer
        , dismissibleDrawerConfig
        , drawerScrim
        )
import Material.List
    exposing
        ( list
        , listConfig
        , listItem
        , listItemConfig
        )

main =
    Html.div []
    [ modalDrawer
        { modalDrawerConfig
            | open = True
            , onClose = Nothing
        }
        [ drawerContent [] [] ]
    , drawerScrim [] []
    , Html.div [] [ text "Main Content" ]
    ]
```

Its 95% stolen from the documentation. Had to fix `onClick` to `onClose` for it to work. But still n Drawer in sight. I'm using 2.1.1 on the elm side as well on the index.js side (using bundler). The only drawer I'm able to get to work is the `permanentDrawer`.

**Comment by j340m3** [comment]
In the end, this was a problem with create-elm-app, which was unable to bind in the js and css files.

Solution:
Against the recommendation in [the create-elm-app readme](https://github.com/halfzebra/create-elm-app/blob/master/template/README.md#changing-the-page-title), pasting the 3 html lines there works, whereas pasting the require lines in index.js has no effect. If anyone can tell me, how this can be solved more elegantly, please let me know.

TLDR; error was located elsewhere.

**Comment by aforemny** [comment]
Hi @j340m3, thank you for your effort in figuring this out! Thank you especially for providing the fix #80. I will merge it shortly!

**Comment by j340m3** [comment]
You're welcome. If you want, I can share my experience on this for #82 . If so, should I wait for your merge or adapt #80 accordingly?
